utility/BUILD.bazel

```diff
@@ -31,10 +33,73 @@ kt_android_library(
     visibility = ["//visibility:public"],
 )
 
-'''
-TODO(#59): Write a kt_android_library() rule to build test files and an android_local_test() rule.
-Because test files depend on the testing module source files being built, these rules have been left
-for a later PR.
-'''
+kt_android_library(
+    name = "utility_tests",
+    custom_package = "org.oppia.util",
+    srcs = glob(["src/test/java/org/oppia/util/**/*.kt"],),
+    resource_files = glob(["src/main/res/**/*.xml"]),
```

Comment:
Are resources actually needed for these tests?

Reply:
I have removed this library since it wasn't being used anymore (per previous PR)